src/test/java/com/sap/oss/phosphor/fosstars/data/github/CodeqlDataProviderTest.java

```diff
@@ -97,7 +97,7 @@ private void testCodeqlRuns(String filename, InputStream content, Value<?>... ex
     CodeqlDataProvider provider = new CodeqlDataProvider(fetcher);
     ValueSet values = provider.fetchValuesFor(PROJECT);
 
-    assertEquals(2, expectedValues.length);
+    assertEquals(2, values.size());
```

Comment:
I guess, this update is not necessary.

Reply:
There is already a Test confirming that the possible features that will be available here https://github.com/SAP/fosstars-rating-core/pull/762/files/f9d5d97620fce7554c112f9a58b257816e56aa96#diff-7567ba8d1e196c5107745af1dcfa55c3f64d94c9b85201c05247500f2c7944a8L59

But you are right. As we are ourselves passing the expected values. The test is the same.